OpenEdXMobile/res/layout/activity_course_base.xml

```diff
@@ -113,46 +113,75 @@
                             android:layout_height="wrap_content"
-                            android:orientation="vertical"
+                            android:layout_weight="1"
+                            android:orientation="horizontal"
                             tools:targetApi="17">
 
+
+                            <org.edx.mobile.view.custom.IconImageViewXml
+                                android:id="@+id/previous_button_icon"
+                                android:layout_width="@dimen/fa_xxx_small"
+                                android:layout_height="@dimen/fa_xx_large"
+                                android:layout_gravity="center_vertical"
+                                android:layout_margin="5dp"
+                                android:baselineAlignBottom="true"
+                                app:iconColor="@color/disableable_button_text"
+                                app:iconName="fa-angle-left" />
+
                             <Button
                                 android:id="@+id/goto_prev"
                                 style="@style/edX.Widget.DisableableButton"
                                 android:layout_width="wrap_content"
-                                android:layout_height="@dimen/course_unit_nav_bar_btn_height"
+                                android:layout_height="match_parent"
                                 android:text="@string/assessment_previous"
+                                android:textAllCaps="true"
                                 tools:targetApi="17" />
 
                             <TextView
                                 android:id="@+id/prev_unit_title"
                                 style="@style/section_lbl"
                                 android:visibility="gone" />
 
-
                         </LinearLayout>
 
-                        <View
-                            android:layout_width="0px"
-                            android:layout_height="0px"
-                            android:layout_weight="1" />
+                        <FrameLayout
+                            android:id="@+id/indicator_container"
+                            android:layout_width="match_parent"
+                            android:layout_height="match_parent"
+                            android:layout_gravity="bottom|center"
+                            android:layout_weight="0.75" />
 
                         <LinearLayout
-                            android:layout_width="wrap_content"
+                            android:layout_width="match_parent"
                             android:layout_height="wrap_content"
+                            android:layout_weight="1"
                             android:gravity="end"
-                            android:orientation="vertical"
+                            android:orientation="horizontal"
                             tools:targetApi="17">
 
                             <Button
```

Comment:
As per discussion next/previous click listeners should be on parent container i.e. LinearLayout.